api/types/events/events.proto

```diff
@@ -1292,6 +1292,36 @@ message BillingCardDelete {
         [ (gogoproto.nullable) = false, (gogoproto.embed) = true, (gogoproto.jsontag) = "" ];
 }
 
+// LockCreate is emitted when a lock is created/updated.
```

Comment:
I think it would be helpful to add a comment stating what the locks are for, ie, what are they be used to lock. Doesn't have to be in-depth, but a few examples would go a long way for someone without previous knowledge.

Reply:
Wouldn't you consider the comments around the `Lock` resource itself (in `types.proto`) sufficient?

Reply:
Is this the one you mean: [LockV2](https://github.com/gravitational/teleport/blob/a177241e89513fb15e6e2e66fb1f40183755ae08/api/types/types.proto#L2255)?

I would be more descriptive there too. To figure out what we're locking one has to go from LockV2 to LockSpecV2 to LockTarget, and even then I'm inferring from the definitions what it can do.

As an example, I would add something like this:

```
// LockV2 represents a lock.
// Locks are created by admins to prevent access, permanently or temporarily, to
// a certain resource, such as a host, user, role, node, etc.
// Refer to LockTarget for the possible targets of a lock.
// Locking is explained by "RFD9 - Locking" (https://github.com/gravitational/teleport/blob/master/rfd/0009-locking.md) (or link some other doc you prefer).
message LockV2 {
  // ...
}
```

This goes a long way for someone trying to understand the system.

In the particular case of LockCreate, I'd be OK  repeating some of that second sentence ("Locks prevent access to a host, user, etc (...)"), while also linking to LockV2 for further reference.

Reply:
Nice, thanks for the changes.